src/Grid.cpp

```diff
@@ -0,0 +1,114 @@
+#include "Grid.hpp"
+#include <fstream>
+#include "common.hpp"
+
+namespace mathboard 
+{
+  Grid::Grid(const cv::Point2f &topLeftCorner, const cv::Point2f &botRightCorner, float cellSize) :
+      m_TopLeftCorner(topLeftCorner), m_BotRightCorner(botRightCorner),
+      m_CellSize(cellSize),
+      m_Height((m_BotRightCorner.y - m_TopLeftCorner.y) / m_CellSize),
+      m_Width((m_BotRightCorner.x - m_TopLeftCorner.x) / m_CellSize) 
+  {
+    if(m_CellSize <= 0) 
+    {
+        std::ofstream debug_stream("debug_output.txt", std::ios::app);
+        debug_stream << "Cell size have to be positive number\n";
```

Comment:
It would be cool if you also added information on where the debug output comes from.
Example:
```cpp
      std::ofstream debug_stream("debug_output.txt",
                                 std::ios::app); // Debug output stream

      debug_stream << "[OpenCVHelper::OpenFile] Error: Could not open video. "
                   << file << std::endl;

      debug_stream.close();
```